src/main/python/janusgraph_python/core/attribute/GeoPredicate/GeoContains.py

```diff
@@ -0,0 +1,55 @@
+"""
+Copyright 2018 JanusGraph Python Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+__author__ = "Debasish Kanhar (https://github.com/debasishdebs)"
```

Comment:
@mbrukman : I would prefer keeping just the author and credits in each file in addition to having AUTHORS.txt. That way we can have who was primary developer for a particular file while having AUTHORS.txt won't help us in pinpointing the same. 

Also, another reason why I'm for having those is that, in future if someone wants to extend the library, they will know the exact person to contact to depending on author rather than contacting everyone on AUTHORS.txt.

I deleted all other fields like email, version etc but kept just author and credits. As for your question is author a single value element, well no, we can keep adding more and more authors as comma separated in same variable

Reply:
@debasishdebs I think all of the `__` fields should be removed. With this PR, you are contributing the code to JanusGraph which is a community-supported project. You are the author of these files, a fact and credit that will never change, and it will be reflected in the GitHub history if someone wants to find out who is the original author.

The best avenue for support is to leverage the public channels (mailing list, issues tracker, etc) so the entire community can continue to support the code, for example if you are too busy or if you are no longer involved with the project. Let's say 5 years from now, the JanusGraph Python driver is wildly popular, but you no longer work on it. You might not want or be able to help developers asking you questions directly about code you developed 5 years ago.